kafka_consumer/datadog_checks/kafka_consumer/client/confluent_kafka_client.py

```diff
@@ -46,23 +51,63 @@ def get_highwater_offsets(self):
                         _, high_offset = consumer.get_watermark_offsets(topic_partition)
 
                         self._highwater_offsets[(topic, topic_partition.partition)] = high_offset
-    
 
     def get_consumer_offsets(self):
```

Comment:
We should probably rename this method because we do not get anything from it. We populate the `self._consumer_offsets` dict

Reply:
After talking on Slack, I think we should actually just return the offsets from this function than a separate `get_*_offsets_dict()` function.